.github/workflows/messaging.yml

```diff
@@ -0,0 +1,76 @@
+name: messaging
+
+on:
+  pull_request:
+    paths:
+    - 'Firebase/Messaging/**'
+
+    #Podspec
+    - 'FirebaseMessaging.podspec'
+
+    - '.github/workflows/messaging.yml'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+  messaging:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+
+    - name: BuildAndTest # can be replaced with pod lib lint with CocoaPods 1.10
+      run: scripts/third_party/travis/retry.sh scripts/build.sh Messaging all
+
+  pod-lib-lint:
+    runs-on: macOS-latest
+
+    strategy:
+      matrix:
+        target: [ios, tvos, macos, watchos]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Build and test
+      if: matrix.target == 'watchos'
+        run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseMessaging.podspec --skip-tests --platforms=${{ matrix.target }}
+      if: matrix.target != 'watchos'
+        run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseMessaging.podspec --platforms=${{ matrix.target }}
+
+  messaging-cron-only:
+    runs-on: macos-latest
+    if: github.event_name == 'schedule'
```

Comment:
To test, comment this out and uncomment when it goes green.

Reply:
comment it out and it went green.